exercise/static/exercise/poll.js

```diff
@@ -73,25 +79,39 @@
 	    },
 
 		message: function(messageType) {
+
 			this.element.removeClass("active").find(this.settings.message_selector)
-				.text(this.element.attr(this.settings.message_attr[messageType]));
-			if (messageType == "error") {
-				this.element.addClass("progress-bar-danger");
-			}
+			.text(this.element.attr(this.settings.message_attr[messageType]));
+			if (this.element.data("aplus-active-element")) {
```

Comment:
Could this be done with `this.element.attr(this.settings.active_element_attr))`?